src/main/viz/PileupCoverageTrack.js

```diff
@@ -65,11 +65,21 @@ class CoverageTiledCanvas extends TiledCanvas {
       .nice();
   }
 
+  localYScaleForRef(ref: string): (y: number) => number {
```

Comment:
I dont love the idea of having YScaleForRef and localYScaleForRef. We should probably just have 1 function.